examples/tenant_policies_ipsla_monitoring_policy/main.tf

```diff
@@ -0,0 +1,53 @@
+terraform {
+  required_providers {
+    mso = {
+      source = "CiscoDevNet/mso"
+    }
+  }
+}
+
+provider "mso" {
+  username = "" # <MSO username>
+  password = "" # <MSO pwd>
+  url      = "" # <MSO URL>
+  insecure = true
+}
+
+data "mso_site" "site_1" {
+  name = "example_site_1"
+}
+
+data "mso_site" "site_2" {
+  name = "example_site_2"
+}
```

Comment:
Is site configuration required for this resource?

Reply:
Nope!